src/v/crypto/ossl_context_service.cc

```diff
@@ -313,12 +320,31 @@ ossl_context_service::ossl_context_service(
     thread_worker, std::move(config_file), std::move(module_path), fips_mode)) {
 }
 
-ss::future<> ossl_context_service::start() { return _impl->start(); }
+ss::future<> ossl_context_service::start() {
+    if (in_rp_fixture_test()) {
+        lg.warn(
+          "Detected RP Fixture test, not initializing OSSL Context service");
+        return ss::make_ready_future();
+    } else {
+        return _impl->start();
+    }
+}
```

Comment:
question: Does this mean that the default context is used in fixture tests? What's the implication of that? What are we not testing, are we sure it's limited to not blocking the reactor during I/O?

Reply:
> Does this mean that the default context is used in fixture tests?

yes

> What's the implication of that? What are we not testing, are we sure it's limited to not blocking the reactor during I/O?

We wouldn't be able to run OpenSSL in FIPS mode in fixture tests, however CDT tests and unit tests will be able to.  I believe I/O, if there is any, will be limited to initialization of the OpenSSL library which would only happen once at the start of the testing application and not in between the different test cases.